### PR TITLE
testing/php7-gearman: new aport

### PR DESCRIPTION
**Pull request by vakartel**
PHP wrapper to libgearman

### CODE REVIEW

File: testing/php7-gearman/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+_pkgreal=gearman
+_prefix=pecl-gearman
+pkgver=2.0.3
+_pkgver=${pkgver/_rc/RC}
+pkgrel=0
+pkgdesc="PHP7 wrapper to libgearman"
+url="https://github.com/wcgallego/pecl-gearman"
+arch="all"
+license="PHP"
+options="!check"
```
Comment (andypost) [inline testing/php7-gearman/APKBUILD]: please add comment why there's no `make NO_INTERACTION=1 REPORT_EXIT_STATUS=1 test`

File: testing/php7-gearman/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+install=
+subpackages=
+source="https://github.com/wcgallego/$_prefix/archive/$_pkgreal-$_pkgver.zip"
+builddir="$srcdir/$_prefix-$_pkgreal-$_pkgver"
+
+prepare() {
+	default_prepare
+
+	cd "$builddir"
+	phpize7
```
Comment (andypost) [inline testing/php7-gearman/APKBUILD]: Why this moved to prepare instead of `build()`?

File: testing/php7-gearman/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
+pkgname=php7-gearman
+_pkgreal=gearman
+_prefix=pecl-gearman
+pkgver=2.0.3
+_pkgver=${pkgver/_rc/RC}
+pkgrel=0
+pkgdesc="PHP7 wrapper to libgearman"
+url="https://github.com/wcgallego/pecl-gearman"
```
Comment (andypost) [inline testing/php7-gearman/APKBUILD]: I think better to point to common http://pecl.php.net/package/gearman

Comment (zehweh) [reply testing/php7-gearman/APKBUILD]: http://pecl.php.net/package/gearman is abandoned

Comment (andypost) [reply testing/php7-gearman/APKBUILD]: Nica catch! missed this point

File: testing/php7-gearman/APKBUILD (hunk 1 above)
Comment (andypost) [reply testing/php7-gearman/APKBUILD]: There's tests https://github.com/wcgallego/pecl-gearman/tree/master/tests

**Comment by ncopa** [comment]
Lets continue this in #4036. Thanks!

**Pull request closed without merging**